target/linux/ath79/dts/qca9558_sophos_apxx.dtsi

```diff
@@ -0,0 +1,171 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/*
+ * The hardware of this board family is most likely shared with other devices
+ * from other manufacturers. Edimax appear to be the actual OEM.
+ *
+ * Sophos use the same exact board for the AP55C/AP100C, and AP55/AP100.
+ * Yes, this means your AP55C is a 3x3 AP with a software lock, and your
+ * AP55 is an AP100 with one missing antenna pigtail.
+ *
+ * AP55 and AP55C boards have different physical layouts, but are logically
+ * almost identical. AP55/100 have a USB port which is DNF on the C models.
+ *
```

Comment:
Remove this "empty" line, so the comment is ended directly with "*/" here.

Reply:
Will do.